projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -44,7 +51,8 @@ protected function __construct() {
 		add_action( 'admin_enqueue_scripts', array( $this, 'dequeue_scripts' ), 20 );
 		add_action( 'rest_request_before_callbacks', array( $this, 'rest_api_init' ), 11 );
 
-		$this->domain = ( new Status() )->get_site_suffix();
+		$this->domain          = ( new Status() )->get_site_suffix();
+		$this->$customize_slug = 'customize.php';
```

Comment:
Looks like there's a `$` too much.
```suggestion
		$this->domain         = ( new Status() )->get_site_suffix();
		$this->customize_slug = 'customize.php';
```

Reply:
WoW, that was too simple and obvious.. I guess it was late for me.. 😜